### PR TITLE
Refs #811: Added a couple Tests to AccumuloOutputFormat 

### PR DESCRIPTION
**Pull request by Manno15**
I added createTables test and simulationMode test, which are two of the requested tests to be added in issue #811 .  I am currently working on the rest but I wanted some feedback on how to improve these tests.

### CODE REVIEW

**Comment by Manno15** [comment]
Any input on what I can do to verify these tests better is much appreciated.

**Comment by ctubbsii** [comment]
I'm not sure I see the value in simulation mode at all. Rather than create a test for it, I think it'd be better to not carry it over to this new MapReduce API at all. If we need it, we can always add it later.

**Comment by Manno15** [comment]
Yeah, I agree. The concept of it is nice to have but I couldn't find a good use for it nor could I find a good way to test it.

**Comment by milleruntime** [comment]
@Manno15 @ctubbsii  I will go ahead and drop the simulation mode from the new MR API.

**Comment by milleruntime** [comment]
@Manno15 I removed the simulation mode from the MR API so if you remove that test, this PR should be good to go!

**Comment by Manno15** [comment]
I will push the change first thing tomorrow. I also have the same test for mapreduce/AccumouloOutputFormat ready to go as well which I will push out too.

**Comment by Manno15** [comment]
@milleruntime I pushed my commit to remove SimulationMode from the tests. Does everything else look good? 
I also have the clientPropsPath test ready to go. Should I create a new pull request for that one or just push it onto this branch?

**Comment by milleruntime** [comment]
> @milleruntime I pushed my commit to remove SimulationMode from the tests. Does everything else look good?

Yep looks good.

> I also have the clientPropsPath test ready to go. Should I create a new pull request for that one or just push it onto this branch?

Sure go ahead.  It seems related enough to commit with these changes.  If you push it up today I can merge this PR.

**Comment by milleruntime** [comment]
@Manno15 does this sounds OK for your commit message:
"Added a couple Tests to AccumuloOutputFormat. Closes #811"

**Comment by Manno15** [comment]
@milleruntime There are still two input format builder tests that still need to be added and are currently being worked on. So my commit shouldn't close #811. Besides that, it sounds good.

**Comment by milleruntime** [comment]
OK. Just let me know.

**Comment by Manno15** [comment]
@milleruntime Pushed my tests.

File: hadoop-mapreduce/src/test/java/org/apache/accumulo/hadoop/mapred/AccumuloOutputFormatTest.java [hunk 1]
```diff
@@ -91,4 +93,43 @@ public void testJobStoreException() throws Exception {
...
+  }
+
+  @Test
+  public void testClientPropertiesPath() throws Exception {
+    JobConf job = new JobConf();
+    Properties cp = Accumulo.newClientProperties().to("test", "zk").as("blah", "blah").build();
+
+    try {
+      File file = File.createTempFile("accumulo-client", ".properties", null);
+      file.deleteOnExit();
```
Comment (milleruntime) [inline hadoop-mapreduce/src/test/java/org/apache/accumulo/hadoop/mapred/AccumuloOutputFormatTest.java]: This is cool!

**Pull request merged**